src/renderer/components/ft-select/ft-select.vue

```diff
@@ -1,8 +1,10 @@
 <template>
   <div class="select">
     <select
+      :id="placeholder"
```

Comment:
This solution works well. Great job! One minor change that I'd suggest is doing a regex on the placeholder for ":id"/":for" to remove the whitespace as html ids shouldn't contain whitespace ex: `placeholder.replace(/ /g, '') `

Reply:
This is something I did not know! It is done.